modules/check_in/spec/request/v2/demographics_request_spec.rb

```diff
@@ -5,24 +5,39 @@
 RSpec.describe 'V2::Demographics', type: :request do
   let(:id) { '5bcd636c-d4d3-4349-9058-03b2f6b38ced' }
   let(:memory_store) { ActiveSupport::Cache.lookup_store(:memory_store) }
-  let(:session_params) do
-    {
-      params: {
-        session: {
-          uuid: id,
-          last4: '5555',
-          last_name: 'Johnson'
-        }
-      }
-    }
-  end
 
   before do
     allow(Rails).to receive(:cache).and_return(memory_store)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_enabled').and_return(true)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_enabled', anything).and_return(true)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_logging_enabled').and_return(true)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_custom_cookie_for_low_auth').and_return(true)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_lorota_security_updates_enabled').and_return(false)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_lorota_401_mapping_enabled').and_return(true)
+    allow(Flipper).to receive(:enabled?).with('check_in_experience_set_pre_checkin_status').and_return(true)
```

Comment:
can we please put in backlog tickets to deprecate some of the flags that are no longer needed?